index.ts

```diff
@@ -186,6 +189,19 @@ export function createProxyCache<
     },
   };
 
+  const fetchGuild = async (id: bigint) => {
+    // Return nothing if we don't want fetching
+    if (!options.fetchGuildIfMissing) return null;
+    // Fetch Guild through Helpers
+    const guild = await bot.helpers.getGuild(id);
```

Comment:
Please add a check, whether the guild is unavailable before and then fetch it.

Reply:
Will add, just wondering why does it make a difference? Does a guild being unavailable mean the API wont return it?
@itohatweb